.eslintrc.js

```diff
@@ -23,16 +22,15 @@ module.exports = {
     ecmaVersion: 13,
     sourceType: "module"
   },
-  plugins: ["react", "@typescript-eslint", "mocha", "cypress"],
```

Comment:
Given that Que. has mocha tests already built, is there a more compelling reason to switch to jest?

Reply:
Unfortunately, the mocha tests previously built were not in a useable state. They seemed to have been abandoned mid-way through a large refactor. As a result, I was unable to re-use them. They were also designed to make calls to the dev instance of KeyCloak, which is not ideal.

While both test frameworks are good choices, I prefer Jest, especially when it comes to mocking external dependencies. Jest has better performance, and is generally more lightweight, which I think makes it a better choice here IMO. Given the above and that we didn't have a sunken cost into Mocha due to the previous tests being unusable, I opted to use Jest. I am open to switching back if there are advantages I am not aware of though.